pages/partners/farmlands.vue

```diff
@@ -0,0 +1,126 @@
+<template>
+  <main>
+    <div
+      class="bg-cover bg-[#174B33] bg-blend-overlay bg-[url('~/assets/images/countryside.png')]"
+    >
+      <div class="max-w-6xl mx-auto">
+        <div class="text-content-on-color text-base leading-6 uppercase font-semibold tracking-wider pt-16 md:pt-32 pl-4 md:pl-40 pb-3">
+          Project
+        </div>
+        <div class="text-content-inverse-primary text-4xl leading-10 font-extrabold px-4 md:px-40 py-3">
+          Farmlands
```

Comment:
Perhaps we could steal the HTML from the guides template for this area?

Reply:
@AndyClifford @MeganSteenkamp Just bumping this - can you confirm that we're reusing as much HTML from Tailwind UI as possible? Then I'll be happy to resolve this comment 👍 

Just want to make sure that our pages are consistent and use their best practices

Reply:
@tom-blake I've updated the html to use a `header` tag like we have used in the guides template 👍